package.js

```diff
@@ -19,6 +19,15 @@ Package.onUse(function(api) {
   // font files
   var fontDirectory = 'bower_components/materialize/';
   var fontFiles = [
+    'font/material-design-icons/Material-Design-Icons.eot',
```

Comment:
At least they are not yet deprecated in materializecss and they deliver them still. So in this case that also the mdi-* rules are still in the css I would prefer for backward compatibility to deliver the font until materializecss drop it too